docs/requirements.md

```diff
@@ -15,11 +27,6 @@ installation and defining the number of concurrent analysis.
 Without accounting for analysis ([next section](#analysis)),
 you should need at least the following resources:
```

Comment:
Remove this paragraph as well since we no longer provide the values for vCPUs and memory?